resources/js/login/login.js

```
@@ -1,937 +1,991 @@
+    switch (error_code) {
+      case MFA_ERROR_CODE.MFA_CHALLENGE_REQUIRED:
+        this.setState((prevState) => ({
+          ...prevState,
+          authFlow: FLOW.MFA,
+          disableInput: false,
+          otpLength: otp_length ?? prevState.otpLength,
+          otpLifetime: otp_lifetime ?? prevState.otpLifetime,
+        }));
+        break;
+      default:
+        const redirect = finalUrl && status === HTTP_CODES.OK;
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)